app/routes/_index/index.module.scss

```diff
@@ -38,3 +38,7 @@
     font-size: clamp(60px, 6vw, 110px);
     text-transform: uppercase;
 }
+
+.textOverlayCardLinkLabel {
```

Comment:
`LabelWithArrow` `.root` has `width: 220px` and its scss file is imported after _common.scss_, so I cannot override it from _common.scss_ style.